infrastructure/nomad-configuration/client.tpl.hcl

```diff
@@ -22,5 +22,5 @@ consul {
 telemetry {
   publish_allocation_metrics = true
   publish_node_metrics       = true
-  statsd_address = "${nomad_lead_server_ip}:8125"
+  statsd_address = "${nomad_server_address}:8125"
```

Comment:
There are two values in the TF with the same value, I'd suggest we collapse to just one, and that `nomad_lead_server_ip` is more accurate and specific.

Reply:
I'm a bit confused about this comment because there is only one variable available to this template and it's defined here: https://github.com/AlexsLemonade/refinebio/blob/dev/infrastructure/instances.tf#L214.

However I definitely can rename this variable to keep things more consistent TF-wide.